<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>day_type</class>
 <widget class="QDialog" name="day_type">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date insertion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #8BC6E2;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QButtonBox {
    background-color: #f0f0f0; /* Set the background color for the entire box */
    border: 1px solid #ccc; /* Set a border for the entire box */
    padding: 10px; /* Set padding for the entire box */
}

/* Style the individual QPushButton elements inside the QButtonBox */
QPushButton {
    background-color: orange; /* Set button background color */
    color: black; /* Set button text color */
    border-radius: 15px; /* Set button border radius */
    padding: 10px 20px; /* Set button padding */
    font-size: 16px; /* Set button font size */
    font-family: &quot;Arial Narrow&quot;, sans-serif; /* Set button font family */
}
QPushButton:pressed {
    background-color: darkorange; /* Change color when button is pressed */
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Day description:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="day_desc_textedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    color: black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	background-color:orange;
	color:black;
}</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Day type:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="day_type_textedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    color: black;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>day_type</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>day_type</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
